modules/codemod/lib/v7/spec/updateTabs.spec.ts

```diff
@@ -5,7 +5,7 @@ import {stripIndent} from 'common-tags';
 const expectTransform = expectTransformFactory(transform);
 
 describe('updateModelSignatures', () => {
-  it('should replace the "name" attribute in Tabs.Items with "data-id"', () => {
+  it('should replace the "name" attribute in Tabs.Items and Tabs.Panel with "data-id"', () => {
```

Comment:
`Tabs.Panel` test with `name` and `data-id` is a separate test below.
```suggestion
  it('should replace the "name" attribute in Tabs.Items with "data-id"', () => {
```